permission_handler/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 6.0.0-nullsafety.0
```

Comment:
Please remove the trailing `.0` as we are following the pattern of not adding a `.0` for the first `nullsafety` release.
```suggestion
# 6.0.0-nullsafety

```